### PR TITLE
Add more test cases to bulk skip test

### PR DESCRIPTION
**Pull request by keith-turner**
Adds tablet gaps of size 1,2,3,...,9 tablets to the bulk import load plan in the skip test. The goal of this is to attempt to cover different edge case w/ the bulk import skip property.  Like when the skip property is set to 4 the load plan will have gaps of 3,4,5 tablets that is has to deal with.

### CODE REVIEW

File: test/src/main/java/org/apache/accumulo/test/functional/BulkNewMetadataSkipIT.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ public void setupCluster() throws Exception {
   }
 
   @ParameterizedTest
-  @ValueSource(ints = {0, 0, 2, 4, 8, 16, 32, 64, 128})
+  @ValueSource(ints = {0, 1, 2, 4, 8, 16, 32, 64, 128})
```
Comment (dlmarion) [inline test/src/main/java/org/apache/accumulo/test/functional/BulkNewMetadataSkipIT.java:112]: Just a heads up that the reason for the two zero values was because there might have been an initial startup cost in the first test run. If you are looking at timing information then the first run could be misleading. No issue here changing it to a `1`, just wanted to provide some background.

**Review by dlmarion — APPROVED** [review]

File: test/src/main/java/org/apache/accumulo/test/functional/BulkNewMetadataSkipIT.java [hunk 2]
```diff
@@ -153,6 +153,15 @@ public void test(int skipDistance) throws Exception {
...
 
+    // create data that skips 0,1,2,..,9 tablets
+    int[] h5Rows = new int[] {50, 50 + 1, 50 + 1 + 2, 50 + 1 + 2 + 3, 50 + 1 + 2 + 3 + 4,
+        50 + 1 + 2 + 3 + 4 + 5, 50 + 1 + 2 + 3 + 4 + 5 + 6, 50 + 1 + 2 + 3 + 4 + 5 + 6 + 7,
+        50 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8, 50 + 1 + 2 + 3 + 4 + 5 + 6 + 7 + 8 + 9};
+    String h5 = writeNonContiguousData(fs, dir + "/f5.", aconf, h5Rows);
+    for (int i : h5Rows) {
+      hashes.get(row(i)).add(h5);
+    }
+
```
Comment (ctubbsii) [inline test/src/main/java/org/apache/accumulo/test/functional/BulkNewMetadataSkipIT.java:164]: This could be more succinctly written as:

```java
    var gap = new AtomicInteger(0);
    int[] h5Rows =
        IntStream.iterate(50, i -> gap.get() < 10, i -> i + gap.incrementAndGet()).toArray();
```

The iterate predicate could also be `i -> i < 100` in this case, but if your goal is to track the gap size, it makes more sense to make sure the gap size is smaller than 10.
